### ISSUE TITLE
Ensure plugin files are not served by webserver

### ISSUE THREAD

**Issue by jkva** [coyote-team/coyote-wp#112]
This should be achieved via checking for a `WPINC` environment variable. Ideally it is achieved via a method which does not require a side-effect.

**Comment by jkva** [comment]
This was addressed by [f92cbd8](https://github.com/coyote-team/coyote-wp/pull/138/commits/f92cbd8ab49eb5f6918e4a7233ad0bf5e6d2054e) in #138.
